Meta/build-image-qemu.sh

```diff
@@ -1,17 +1,4 @@
-#!/bin/sh
-
-# Note: This is done before `set -e` to let `command` fail if needed
-FUSE2FS_PATH=$(command -v fuse2fs)
-RESIZE2FS_PATH=$(command -v resize2fs)
-
-if [ -z "$FUSE2FS_PATH" ]; then
-    FUSE2FS_PATH=/usr/sbin/fuse2fs
-fi
-
-if [ -z "$RESIZE2FS_PATH" ]; then
-    RESIZE2FS_PATH=/usr/sbin/resize2fs
-fi
-
+#!/usr/bin/env bash
```

Comment:
This change would also be required for the remaining `build-image-*.sh` scripts and `build-native-partition.sh`, all of those also include `.shell_include.sh`.